src/App.tsx

```diff
@@ -1,46 +1,34 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import React from 'react';
+import { Tabs } from './components/Tabs';
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">Selected tab is Tab 1</h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">
-                Tab 1
-              </a>
-            </li>
+  const [selectedTabId, setSelectedTabId] = React.useState(tabs[0].id);
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">
-                Tab 2
-              </a>
-            </li>
+  const onTabSelected = (tab: (typeof tabs)[0]) => {
+    setSelectedTabId(tab.id);
+  };
 
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">
-                Tab 3
-              </a>
-            </li>
-          </ul>
-        </div>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is {tabs.find(tab => tab.id === selectedTabId)?.title}
```

Comment:
Let's extract this into a separate constant outside the render function, just to make code cleaner.